src/backend/utils/storage.js

```diff
@@ -1,16 +1,38 @@
+const crypto = require('crypto');
+const normalizeUrl = require('normalize-url');
+const { logger } = require('./logger');
 const { redis } = require('../lib/redis');
 
+const standardize = (url, type) => {
+  let namespace = 't:post:';
+  let processed = url;
+
+  if (type === 'feed') {
+    namespace = 't:feed:';
+    processed = normalizeUrl(url);
+  }
+
+  try {
+    return namespace.concat(
+      crypto
+        .createHash('sha256')
+        .update(processed)
+        .digest('base64')
+    );
+  } catch (error) {
+    logger.error(`There was an error processing ${url}`);
+    throw error;
+  }
+};
+
 // Redis Keys
-const FEED_ID = 'feed_id';
 const FEEDS = 'feeds';
```

Comment:
Based on what we're doing with namespaces, this should probably become `t:feeds` and `t:posts` below.

Reply:
The `FEEDS` and `POSTS` keys still need namespacing `t:...`